app/views/admin/administrators/administrators/index.html.erb

```diff
@@ -12,7 +12,7 @@
       <tbody class="govuk-table__body">
         <% @administrators.each do |administrator| %>
           <tr class="govuk-table__row">
-            <td class="govuk-table__cell"><%= administrator.full_name %></td>
+            <td class="govuk-table__cell"><%= govuk_link_to administrator.full_name, edit_admin_administrator_path(administrator) %></td>
```

Comment:
Are we not using the helper here? Why only in some places?

Reply:
I don't think it's strictly needed. My main motivation for the helper was on the suppliers page where we have a list that mixes up delivery partner users and lead provider users and I needed to send each user to the right endpoint. The admins and induction coordinators don't get combined that why I haven't used it here. @ethanmills

Reply:
Fair enough. You do use it for induction coordinators though, right? https://github.com/DFE-Digital/early-careers-framework/pull/122/files#diff-9e81e35702dad35ed3b19ae78f92ad76e63185d92a3158b0818ba73110b36927R14

Reply:
Ok 🙂 Happy for this to go in as-is